dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report.controller/index.test.ts

```diff
@@ -47,13 +47,14 @@ describe('getUtilisationReport', () => {
     });
 
     const mockDate = new Date('2024-01');
+    const epochDate = new Date('01-01-1970');
 
     const createMockIncompleteUploadedReport = (...missingFields: UploadedReportFields[]): UtilisationReportEntity =>
       UtilisationReportEntityMockBuilder.forStatus(status)
         .withId(reportId)
         .withAzureFileInfo(azureFileInfo)
-        .withDateUploaded(missingFields.includes('dateUploaded') ? null : mockDate)
-        .withUploadedByUserId(missingFields.includes('uploadedByUserId') ? null : uploadedByUser.id)
+        .withDateUploaded(missingFields.includes('dateUploaded') ? epochDate : mockDate)
```

Comment:
@ChristianMcc could you please verify this, I believe my approach is a mere workaround since I am not well versed with the utilisation report controller any suggestion on this will be great. Thanks

Reply:
Seems odd to change "if X, pass `null`" to  "if X, pass `epochDate` - i'm not familiar with this test or the functionality, but i'd expect this change to cause the test to fail

Reply:
I imagine this won't cause it to fail, as a report that hasn't been uploaded will ignore date uploaded (as that field shouldn't be there). These are nullable columns and fields so how come we need to default them to something rather than just leaving it as null?

Reply:
Good question @ChristianMcc after regenerating on `package-lock.json` the above error populated during a typescript check. I can further look into the controller as I fear a minor adjustment will be required to the controller as the test is failing due to static type error.

Reply:
> I imagine this won't cause it to fail, as a report that hasn't been uploaded will ignore date uploaded (as that field shouldn't be there). These are nullable columns and fields so how come we need to default them to something rather than just leaving it as null?

Maybe could just remove the `withDateUploaded` line/method in this particular unit test @abhi-markan ?

Reply:
Have updated the argument type as due to the current tests and implementation it is not possible for the test to pass with `null` argument without changing the conditional typing.

I am surprised this was not flagged in the pipeline as either the static typing will not suffice the conditional typing due to `REPORT_NOT_RECONCILE` not present the status array or the existing controller implementation will fail due to incorrect report status.